Makefile

```diff
@@ -151,23 +170,24 @@ package-linux: GOOS:=linux
 package-linux: GOOS_DIR:=$(BUILD_DIR)/$(GOOS)
 package-linux: ARCHIVE_NAME=lifecycle-v$(LIFECYCLE_VERSION)+$(GOOS).x86-64
 package-linux:
-	@echo "> Writing descriptor file for $(GOOS)..."
-	mkdir -p $(GOOS_DIR)
-	echo "$${LIFECYCLE_DESCRIPTOR}" > $(GOOS_DIR)/lifecycle.toml
-
 	@echo "> Packaging lifecycle for $(GOOS)..."
-	tar czf $(BUILD_DIR)/$(ARCHIVE_NAME).tgz -C $(GOOS_DIR) lifecycle.toml lifecycle
+	$(GOCMD) run tools/packager/main.go -os $(GOOS) -launcherExePath $(GOOS_DIR)/lifecycle/launcher -lifecycleExePath $(GOOS_DIR)/lifecycle/lifecycle -lifecycleVersion $(LIFECYCLE_VERSION) -platformAPI $(PLATFORM_API) -buildpackAPI $(BUILDPACK_API) -outputPackagePath $(BUILD_DIR)/$(ARCHIVE_NAME).tgz
 
-package-windows: export LIFECYCLE_DESCRIPTOR:=$(LIFECYCLE_DESCRIPTOR)
 package-windows: GOOS:=windows
-package-windows: GOOS_DIR:=$(BUILD_DIR)/$(GOOS)
+package-windows: GOOS_DIR:=$(BUILD_DIR)$/$(GOOS)
```

Comment:
What's the `$` for? Is it a typo?

Reply:
It's not actually. It's an OS-agnostic slash. You'll see the definition near the top of the file :)

Reply:
In this case, the `$/` allows the same target to work for both `bash` and `cmd` since the syntax is otherwise the same. My hope is that while writing cross-shell syntax may be a little trickier, by keeping most targets as identical as possible it will make it less likely for Windows and Linux coverage to diverge.

Reply:
Ah! That is fancy...